web/src/App.tsx

```diff
@@ -19,8 +20,22 @@ class App extends Component {
   constructor(props: any) {
     super(props);
     this.state = {
-      modal: false,
+      modal: '',
       request: {},
+      tables: [
+        {
+          source: 'ViewTable1',
+          rows: [],
+          fields: ['Num1', 'Num2', 'I'],
+          headers: ['ID', 'Num1', 'Num2', 'I'],
+        },
+        {
+          source: 'ViewTable2',
+          rows: [],
```

Comment:
now don't use, but in future I think state ` ViewTable1: [],` will be here

Reply:
Let's bring back the hard-coded values, please :smile: